.ado/templates/publish-nuget-job.yml

```diff
@@ -0,0 +1,21 @@
+steps:
+  # generate the sanitizedBuildNumber environment variable
+  - bash: 'scripts/generate_build_number.sh'
+    displayName: 'Generate Build Number'
+
+  # pack the nuspec
+  - task: NuGetCommand@2
+    displayName: 'NuGet pack'
+    inputs:
+      command: pack
+      packagesToPack: 'FluentUI.nuspec'
```

Comment:
This should probably be FluentUI.ReactNative.nuspec or something else to distinguish from other FluentUI?

Reply:
the name doesn't actually matter. The id inside the nuspec matters, which is Microsoft.FluentUI.ReactNative

I'll probably rename this package.nuspec just to be more generic